pkg/app/pipectl/cmd/deployment/list.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2023 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package deployment
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/pipe-cd/pipecd/pkg/app/server/service/apiservice"
+	"github.com/pipe-cd/pipecd/pkg/cli"
+	"github.com/pipe-cd/pipecd/pkg/model"
+)
+
+type list struct {
+	root *command
+
+	statuses []string
+	appKinds []string
+	appIds   []string
+	appName  string
+	labels   []string
+	limit    int32
+
+	cursor string
+	stdout io.Writer
+}
+
+func newListCommand(root *command) *cobra.Command {
+	c := &list{
+		root:   root,
+		stdout: os.Stdout,
+	}
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Show the list of deployments.",
+		RunE:  cli.WithContext(c.run),
+	}
+
+	cmd.Flags().StringSliceVar(&c.statuses, "status", c.statuses, fmt.Sprintf("The list of waiting statuses. (%s)", strings.Join(model.DeploymentStatusStrings(), "|")))
+	cmd.Flags().StringSliceVar(&c.appIds, "app-id", c.appIds, fmt.Sprintf("The application id. (%s)", strings.Join(model.ApplicationKindStrings(), "|")))
+	cmd.Flags().StringSliceVar(&c.appKinds, "app-kind", c.appKinds, fmt.Sprintf("The kind of application. (%s)", strings.Join(model.ApplicationKindStrings(), "|")))
```

Comment:
```suggestion
	cmd.Flags().StringSliceVar(&c.statuses, "statuses", c.statuses, fmt.Sprintf("The list of waiting statuses. (%s)", strings.Join(model.DeploymentStatusStrings(), "|")))
	cmd.Flags().StringSliceVar(&c.appIds, "app-ids", c.appIds, fmt.Sprintf("The application ids. (%s)", strings.Join(model.ApplicationKindStrings(), "|")))
	cmd.Flags().StringSliceVar(&c.appKinds, "app-kinds", c.appKinds, fmt.Sprintf("The kinds of applications. (%s)", strings.Join(model.ApplicationKindStrings(), "|")))
```

Reply:
I'm making it singular because it doesn't necessarily have to be entered in the plural and the statuses argument in deployment/waitstatus.go was singular, what do you think?

Reply:
It's okay if you are doing so intentionally and it's up to you. 
The important things are whether easy to know about being able to specify multiple values.

Reply:
I see.
I'll try to make it a little clearer.

Reply:
I revised it. How about it?